src/routes/v2/authenticatedSites/index.js

```diff
@@ -196,6 +200,10 @@ const getAuthenticatedSitesSubrouter = ({
   const navigationV2Router = new NavigationRouter({
     navigationYmlService: navYmlService,
   })
+  const repoManagementV2Router = new RepoManagementRouter({
+    repoManagementService,
+    authorizationMiddleware,
```

Comment:
how is the FE cypress run able to authorise itself to this route?

Reply:
This happens in a series of events:
1. The frontend sets a special cookie that is recognised by the backend via the authentication middleware. When this cookie is present, the authentication middleware will set a series of defaults to "become" a special e2e user. This can be either the GitHub user (which is currently our isomeradmin default user), or the e2e email login user.
2. With this user session set up, the authorization middleware will take over to determine if the user is authorised to access the site. There's no special treatment here, as the special e2e users have access to the e2e test repos.
3. Finally, at this router level, the access would have been granted, so the frontend will be able to enter this route and access the repo management routes.

Reply:
Got it, so is a agreed upon cookie string